statement.go

```diff
@@ -186,6 +204,28 @@ func (s *Stmt) bindTime(val driver.NamedValue, t Type, n int) (mapping.State, er
 	return state, nil
 }
 
+func toAnySlice(v any) ([]any, error) {
+	// If already []any, return as is
+	if slice, ok := v.([]any); ok {
+		return slice, nil
+	}
+
+	// Use reflection to handle other cases
+	rv := reflect.ValueOf(v)
+
+	// If it's a slice or array, convert each element
+	if rv.Kind() == reflect.Slice || rv.Kind() == reflect.Array {
+		result := make([]any, rv.Len())
+		for i := 0; i < rv.Len(); i++ {
+			result[i] = rv.Index(i).Interface()
+		}
+		return result, nil
+	}
+
+	// Handle single values by wrapping in a slice
+	return []any{v}, nil
```

Comment:
I prefer if we return an invalid input error here, as strictly speaking the user did not provide an `ARRAY` or `LIST` as a parameter.